### PR TITLE
Add new SQLite Math function mappings

### PR DESCRIPTION
**Pull request by bricelam**
Part of #4419

~~Blocked waiting for dotnet/efcore#31356~~

### CODE REVIEW

File: entity-framework/core/providers/sqlite/functions.md [hunk 1]
```diff
@@ -97,29 +97,79 @@ dateTime.Year                   | strftime('%Y', @dateTime)
...
+Math.Round(d)                    | round(@d)
+Math.Round(d, digits)            | round(@d, @digits)
+Math.Sign(d)                     | sign(@d)                              | EF Core 8.0
+Math.Sin(a)                      | sin(@a)                               | EF Core 8.0
+Math.Sinh(value)                 | sinh(@value)                          | EF Core 8.0
+Math.Sqrt(d)                     | sqrt(@d)                              | EF Core 8.0
+Math.Tan(a)                      | tan(@a)                               | EF Core 8.0
+Math.Tanh(value)                 | tanh(@value)                          | EF Core 8.0
+Math.Truncate(d)                 | trunc(@d)                             | EF Core 8.0
+MathF.Acos(x)                    | acos(@x)                              | EF Core 8.0
```
Comment (roji) [inline entity-framework/core/providers/sqlite/functions.md:148]: Maybe it's worth just adding an asterisk and a footnote, saying that all corresponding MathF methods are translated as well, to not make this needlessly long.

**Review by roji — APPROVED** [review]

File: entity-framework/core/providers/sqlite/functions.md (hunk 1 above)
Comment (bricelam) [reply entity-framework/core/providers/sqlite/functions.md:148]: Yeah, I like it. I was also going to add a note that corresponding generic math methods are also translated, so I'll do both at the same time.

File: entity-framework/core/providers/cosmos/functions.md [hunk 2]
```diff
@@ -44,23 +42,11 @@ Math.Sin(a)                | SIN(@a)
...
-MathF.Log10(x)             | LOG10(@x)
-MathF.Pow(x, y)            | POWER(@x, @y)
-MathF.Round(x)             | ROUND(@x)
-MathF.Sin(x)               | SIN(@x)
-MathF.Sqrt(x)              | SQRT(@x)
-MathF.Tan(x)               | TAN(@x)
-MathF.Truncate(x)          | TRUNC(@x)
+
+> [!TIP]
+> In addition to the methods listed here, corresponding [generic math](/dotnet/standard/generics/math) implementations
```
Comment (roji) [inline entity-framework/core/providers/cosmos/functions.md:47]: :+1:

Can put links to the MathF class docs and to the different functions if you want.

**Review by roji — APPROVED** [review]

File: entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 
-This page will document API and behavior changes that have the potential to break existing applications updating to EF Core 8.0.
+This page documents API and behavior changes that have the potential to break existing applications updating to EF Core 8.0.
 
 ## Summary
 
 | **Breaking change**                                                                                                                      | **Impact** |
 |:---------------------------------------------------------------------------------------------------------------------------------------- | ---------- |
 | [SQL Server `date` and `time` now scaffold to .NET `DateOnly` and `TimeOnly`](#sqlserver-date-time-only)                                 | Medium     |
+| [SQLite `Math` methods now translate to SQL](#sqlite-math)                                                      | Low        |
```
Comment (roji) [inline entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md:18]: nit: table alignment

**Review by roji — APPROVED** [review]

File: entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md (hunk 3 above)
Comment (bricelam) [reply entity-framework/core/what-is-new/ef-core-8.0/breaking-changes.md:18]: Oops, thanks. Re-worded after my initial formatting

**Pull request merged**